CHANGELOG.md

```diff
@@ -6,7 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [Unreleased] 1.8.0
+
+### Changed
+
+- Enable `SIGNALFX_SERVER_TIMING_CONTEXT` by default. ([#96](https://github.com/signalfx/signalfx-go-tracing/pull/96))
```

Comment:
In theory, this is a breaking change but I think bumping to `v2` does not make sense in this case.
@MrAlias @owais  Do you agree?

Reply:
I agree. It's debatable whether this is a breaking change or not. As long as we don't overwrite any possible headers set by the user, I wouldn't call this a breaking change.